tests/test_cli/test_inputhandler/test_inputhandler.cpp

```diff
@@ -397,22 +397,16 @@ TEST(PemFileReader, getPublicPrivateKeys_expectSameResultFrom_libsodium)
 {
     ih::PemFileReader pemHandler("..//testData//keys.pem");
 
-    unsigned char pemPk[crypto_sign_PUBLICKEYBYTES];
-    unsigned char pemSk[crypto_sign_SECRETKEYBYTES];
-    unsigned char pemSeed[crypto_sign_SEEDBYTES];
-
-    unsigned char sodiumPk[crypto_sign_PUBLICKEYBYTES];
-    unsigned char sodiumSk[crypto_sign_SECRETKEYBYTES];
-
-
     bytes seedBytes = pemHandler.getSeed();
     bytes skBytes = pemHandler.getPrivateKey();
     bytes pkBytes = pemHandler.getAddress().getPublicKey();
 
-    std::copy(seedBytes.begin(), seedBytes.end(), pemSeed);
-    std::copy(skBytes.begin(), skBytes.end(), pemSk);
-    std::copy(pkBytes.begin(), pkBytes.end(), pemPk);
+    auto pemPk = reinterpret_cast<unsigned char*> (pkBytes.data());
```

Comment:
Please add the "documentation" signing tests. 

https://github.com/ElrondNetwork/elrond-go/blob/master/examples/construction_test.go
https://github.com/ElrondNetwork/elrond-sdk-erdjs/blob/main/src/walletcore/users.spec.ts#L120

(with those values, with that data, and expect those signatures)

Reply:
I've added just one test out of those.
That's because in [this](https://github.com/ElrondNetwork/elrond-sdk-erdcpp/pull/6/files) PR, transaction initialization will be changed, so will need to do lot of refactoring. 
Therefore, I will add all those tests in a future PR